wrapdocker

```diff
@@ -122,9 +122,10 @@ if [ "${STORAGE_DIR_FS}" = "overlay" ] && [ "${STORAGE_FS}" = "overlay" ]; then
   rm -rf "${D}" || true
 fi
 
-# Unless using overlay over overlay or overlay over overlay is broken, create an ext3 loop device as an intermediary layer.
-# The max size of the loop device is $VAR_LIB_DOCKER_SIZE in GB (default=5).
-if [ "${STORAGE_DIR_FS}" != "overlay" ] || [ "${STORAGE_FS}" != "overlay" ] || [ "${OOO_BROKEN}" == true ]; then
+# If another filesystem than overlay is used or overlay does not work, create
+# an ext3 loop device as an intermediary layer. The max size of the loop device
+# is $VAR_LIB_DOCKER_SIZE in GB (default=5).
+if [ "${STORAGE_FS}" != "overlay" -o ${OVERLAY_WORKS} != true ]; then
```

Comment:
prolly want to quote `${OVERLAY_WORKS}` to catch edge cases: http://stackoverflow.com/questions/2953646/how-to-declare-and-use-boolean-variables-in-shell-script

I'm tend to prefer `] || [` over `-o` just to increase readability, but it's not critical.